cmd/podman/networks/connect.go

```diff
@@ -0,0 +1,45 @@
+package network
+
+import (
+	"github.com/containers/podman/v2/cmd/podman/common"
+	"github.com/containers/podman/v2/cmd/podman/registry"
+	"github.com/containers/podman/v2/pkg/domain/entities"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+var (
+	networkConnectDescription = `Add container to a network`
+	networkConnectCommand     = &cobra.Command{
+		Use:               "connect [options] NETWORK CONTAINER",
+		Short:             "network connect",
+		Long:              networkConnectDescription,
+		RunE:              networkConnect,
+		Example:           `podman network connect web secondary`,
+		Args:              cobra.MinimumNArgs(2),
+		ValidArgsFunction: common.AutocompleteNetworks,
+	}
+)
+
+var (
+	networkConnectOptions entities.NetworkConnectOptions
+)
+
+func networkConnectFlags(flags *pflag.FlagSet) {
+	flags.StringSliceVar(&networkConnectOptions.Aliases, "alias", []string{}, "network scoped alias for container")
+}
```

Comment:
This will fail with my new completion validation. Friendly suggestion:
```suggestion
func networkConnectFlags(cmd *cobra.Command) {
	flags := cmd.Flags()
	aliasFlagName := "alias"
	flags.StringSliceVar(&networkConnectOptions.Aliases, aliasFlagName, []string{}, "network scoped alias for container")
	_ = cmd.RegisterFlagCompletionFunc(aliasFlagName, completion.AutocompleteNone)
}
```
completion is imported from `github.com/containers/common/pkg/completion`